anyway/infographics_utils.py

```diff
@@ -234,6 +236,21 @@ def get_accident_count_by_severity(location_info, start_time, end_time):
         items["total_accidents_count"] = total_accidents_count
         return items
 
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        if request_params.lang != "en":
+            try:
+                items["meta"]["information"] = _(items["meta"]["information"])
+            except KeyError:
+                information = items["meta"]["information"]
```

Comment:
If `items["meta"]["information"]` will raise a KeyError exception then there is a risk that it will be raised again inside the the except clause as `items["meta"]["information"]` is also being referenced there.
Do we need the except clause?


I think it is also present in other `localize_items` functions.

Reply:
correct. `items["meta"]["information"]` does not exist, so not point in referencing it.
Do we want to set other value in the exception handling?